### PR TITLE
Merges v1-1 into master

### PR DESCRIPTION
**Pull request by cipherboy**
- Merges v1-1 into master

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,12 +18,36 @@ add_definitions(-DQT_DISABLE_DEPRECATED_BEFORE=0)
 # custom openscap parameters.
 if (NOT DEFINED OPENSCAP_LIBRARIES OR NOT DEFINED OPENSCAP_INCLUDE_DIRS)
     include("${CMAKE_ROOT}/Modules/FindPkgConfig.cmake")
-    pkg_check_modules(OPENSCAP REQUIRED libopenscap>=1.2.11)
+    pkg_check_modules(OPENSCAP REQUIRED libopenscap>=1.2.0)
```
Comment (mpreisler) [inline CMakeLists.txt]: this looks suspect

Comment (cipherboy) [reply CMakeLists.txt]: I think this is legit: https://github.com/OpenSCAP/scap-workbench/commit/a2265f5a44bef600807ca64358cb68d24ceea09e

Comment (mpreisler) [reply CMakeLists.txt]: yeah, it is

**Comment by mpreisler** [comment]
Good job us for not merging into master for so long :-D

**Comment by mpreisler** [comment]
The jenkins fail is expected, qt5 is not on the rhel7 jenkins node yet.

**Pull request merged**